msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -143,13 +144,16 @@ The following image displays the **Contoso** tab with results of poll and feedba
 :::image type="content" source="~/assets/images/apps-in-meetings/mobilepremeeting.png" alt-text="Post meeting app experience.":::
 
 ---
+In scheduled public channel meetings, after a meeting app is added it can be accessed from the meeting details page by selecting on the meeting object. See the following example:
```

Comment:
See my suggestion above to clarify this to "tab", not "meeting app" to avoid conflating the concepts.

Reply:
Updated as mentioned.